mono/metadata/Makefile.am

```diff
@@ -82,6 +82,10 @@ if SUPPORT_BOEHM
 boehm_libraries = libmonoruntime.la
 endif
 
+if ENABLE_CORE
```

Comment:
where is this introduced? missing `configure.ac` changes?

Reply:
It's already there at https://github.com/mono/mono/blob/master/configure.ac#L764